10-setup-htcondor.sh

```diff
@@ -8,8 +8,13 @@ if [ `id -u` = 0 ]; then
 fi
 
 # validation
-if [ "x$TOKEN" = "x" ]; then
-    echo "Please specify TOKEN as an environment variable" 1>&2
+if [[ ! -e /etc/condor/tokens.d/flock.opensciencegrid.org ]] &&
+   [[ ! $TOKEN ]]; then
+    { echo "Please provide /etc/condor/tokens-orig.d/flock.opensciencegrid.org"
+      echo "via volume mount."
+      echo "Alternatively, TOKEN may be provided as an environment variable"
+      echo "Please specify TOKEN as an environment variable"
```

Comment:
```suggestion
```

Let's not advertise the `TOKEN` env var and just grandfather folks in that may be currently using it